students/mielczarek_szymon/lesson_06_dicts_tuples_sets_args_kwargs/countries_and_cities.py

```diff
@@ -0,0 +1,10 @@
+country_dict = {}
+for _ in range(int(input())):
+    country, *cities = input().split()
+    country_dict[country] = ' '.join(cities)
```

Comment:
I see no point in `join` here.
Lets simplify this line to:
`country_dict[country] = cities`